projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -40,3 +40,20 @@ export function verifyTicketDetailsPageVisit(){
   cy.url().should('match',/http:\/\/.+\/my\-account\/support\-ticket\/[0-9]+/);
   cy.get('cx-customer-ticketing-messages').should('exist');
 }
+
+export function visitTicketDetailsPageForFirstTicket(){
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).should('exist');
+  cy.get(`#ticketing-list-table tbody tr:nth-child(${FIRST_TICKET_ROW_INDEX}) .cx-ticketing-list-data:nth-child(${FIRST_TICKET_COLUMN_INDEX}) a.cx-ticketing-list-value`).then( ($ticketIdElement) => {
```

Comment:
same for .then

Reply:
I have some of these defined in the merged e2e and some in the ticket list e2e

Reply:
They are now fixed. Please check and confirm.